src/plugins/auto/tests/unit/parse_meta_device_test.cpp

```diff
@@ -134,6 +136,11 @@ const std::vector<ConfigParams> testConfigs = {
              {"GPU.0", {}, -1, "", std::string(igpuFullDeviceName) + "_0", 1},
              {"GPU.1", {}, -1, "", std::string(dgpuFullDeviceName) + "_1", 1},
              {"CPU", {}, -1, "", "CPU_", 2}}, false},
+    ConfigParams {"VPUX,GPU,INVALID_DEVICE",
+         {{"VPUX", {}, -1, "", "VPUX_", 0},
```

Comment:
In 2023.1 release, we should use "VPU" in test case.

Reply:
I need to rename all of the VPUX to VPU in this file if we changed here to pass this added test case for VPU instead of VPUX.  is this our expected behavior?

Reply:
@wangleis  Being covered by https://github.com/openvinotoolkit/openvino/pull/17949